ctp2_code/ui/interface/diplomacywindow.cpp

```diff
@@ -2969,22 +3008,29 @@ void DiplomacyWindow::TabPanelActionCallback(aui_Control *control, uint32 action
 
 	if(action != static_cast<uint32>(ctp2_Tab::ACTION_ACTIVATED))
 		return;
-	sint32 section = (intptr_t)cookie;
+
+	sint32 section = (sint32)cookie;
+
```

Comment:
some of those blank lines (above and below) seem to be redundant to me, although I admit that this is very subjective (especially in absence of some sort of coding standard)

Reply:
Sure they are, same reaseon as above. (Actually you can remove any new line in C as e.g. in this "International Obfuscated C Code Contest" (2004) winner:
https://www.ioccc.org/2004/anonymous.c
https://www.ioccc.org/2004/anonymous.hint
)